### ISSUE TITLE
CrossEntropyLoss reduction write error

### ISSUE THREAD

**Issue by AutumnZ-94** [michuanhaohao/reid-strong-baseline#205]
should

`loss = (- targets * log_probs).mean(0).sum()`  in [https://github.com/michuanhaohao/reid-strong-baseline/blob/master/layers/triplet_loss.py#L146](url)

be supposed to `loss = (- targets * log_probs).sum(1).mean()`?

**Comment by SuperbTUM** [comment]
@AutumnZ-94 Try this. They should be the same

```python
x = torch.randn(32, 64)

y1 = x.mean(0).sum()
y2 = x.sum(1).mean()

assert torch.allclose(y1, y2)
```
